go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kube-openapi v0.0.0-20200121204235-bf4fb3bd569c // indirect
 	k8s.io/utils v0.0.0-20200229041039-0a110f9eb7ab
-	sigs.k8s.io/controller-runtime v0.5.3
+	sigs.k8s.io/controller-runtime v0.5.5-0.20200529152116-5291db5ab70e
```

Comment:
Will there be an official v0.5.5 release we can point to soon for this?

Reply:
Just released it!